docs/en/observability/apm/otel-direct.asciidoc

```diff
@@ -135,6 +78,71 @@ Other methods, like using the https://github.com/open-telemetry/opentelemetry-co
 but will bypass all of the validation and data processing that the APM Server performs.
 In addition, your data will not be viewable in the {kib} {observability} apps if you use the `elasticsearch` exporter.
 
+[discrete]
+[[apm-instrument-apps-otel]]
+==== Send data from an upstream OpenTelemetry SDK
+
+[NOTE]
+====
+While it's possible to send data directly from an upstream OpenTelemetry SDK to APM Server,
+in many cases you should use the SDK to send data to an OpenTelemetry Collector that processes
+and exports data to APM Server. Read more about when you should use a collector in the
+https://opentelemetry.io/docs/collector/#when-to-use-a-collector[OpenTelemetry documentation].
```

Comment:
>This doc outlines how to send data directly from an upstream OpenTelemetry SDK to APM Server. While this is possible, in many cases you should use the SDK to send data to an OpenTelemetry Collector that processes and exports data to APM Server. Read more about when you should use a collector in the
[OpenTelemetry documentation](https://opentelemetry.io/docs/collector/#when-to-use-a-collector).

Reply:
```suggestion
This doc outlines how to send data directly from an upstream OpenTelemetry SDK to APM Server, which is appropriate during development or if you're monitoring a small-scale application. In many cases, you should use the OpenTelemetry SDK to send data to an OpenTelemetry Collector that processes and exports data to APM Server. Read more about when and how to use a collector in the https://opentelemetry.io/docs/collector/#when-to-use-a-collector[OpenTelemetry documentation].
```